modules/ballerina-builtin/src/main/ballerina/ballerina/regex/natives.bal

```diff
@@ -0,0 +1,37 @@
+package ballerina.regex;
+
+@Description { value: "Represents a Regular expression in ballerina and can perform various Regular expression methods."}
+@Field { value : "regex: Regular expression as a String"}
+public struct Regex {
+   string regex;
+}
+
+@Description { value:"Parse Regular Expression string"}
+@Param { value:"reg: The Regular Expression" }
+public native function <Regex reg> compile ();
```

Comment:
What if package compile fails. It should return an error. isn't it?

Reply:
+1